src/main/scala/org/scalajs/dom/experimental/mediastream/MediaStream.scala

```diff
@@ -626,3 +626,178 @@ trait MediaTrackSupportedConstraints extends js.Object {
   var deviceId: js.UndefOr[Boolean] = js.undefined
   var groupId: js.UndefOr[Boolean] = js.undefined
 }
+
+/**
+ * Represents a source of media data for an HTMLMediaElement object.
+ *
+ * A MediaSource object can be attached to a HTMLMediaElement to be played in the user agent.
+ */
+@js.native
+@JSGlobal
+class MediaSource extends EventTarget {
+  def sourceBueffers: SourceBufferList = js.native
+  def activeSourceBuffers: SourceBufferList = js.native
+  def readyState: ReadyState = js.native
+
+  /**
+   * Allows the web application to set the presentation duration.
+   *
+   * The duration is initially set to NaN when the MediaSource object is created.
+   */
+  var duration: Double = js.native
+
+  var onsourceopen: js.Function1[Event, Any] = js.native
+  var onsourceended: js.Function1[Event, Any] = js.native
+  var onsourceclose: js.Function1[Event, Any] = js.native
+
+  def addSourceBuffer(bufferType: String): SourceBuffer = js.native
+  def removeSourceBuffer(sourceBuffer: SourceBuffer): Unit = js.native
+  def endOfStream(error: js.UndefOr[EndOfStreamError] = ???): Unit = js.native
```

Comment:
```suggestion
  def endOfStream(error: EndOfStreamError = js.native): Unit = js.native
```
better matches the IDL definition.

Reply:
Fixed in https://github.com/scala-js/scala-js-dom/pull/365/commits/f810f35f7005100bb590814cbfd4d248d4da5329